utils/led_error.h

```diff
@@ -0,0 +1,18 @@
+/*
+ * led_error.h
```

Comment:
The error type could be defined inside `led.h` and this file skipped altogether...

Reply:
Okay, sure!